src/exits/utils.py

```diff
@@ -50,11 +49,21 @@ async def send_signature_rotation_requests(
                 failed_endpoints.extend(replicas)
                 continue
             approvals[address] = response
-    try:
-        return process_oracles_approvals(approvals, oracles.validators_threshold)
-    except NotEnoughOracleApprovalsError as e:
-        e.failed_endpoints = failed_endpoints
-        raise
+
+    logger.info(
+        'Fetched oracle approvals for signature update: validators count = %d. '
+        'Received: %d out of %d.',
```

Comment:
Fetched oracle approvals for signatures update of %d validators. Received approvals: %d out of %d.